aiida/orm/implementation/general/code.py

```diff
@@ -87,10 +96,42 @@ def __str__(self):
             else:
                 computer_str = "[unknown]"
 
-        return "{} code '{}' on {}, pk: {}, uuid: {}".format(local_str,
-                                                             self.label,
-                                                             computer_str,
-                                                             self.pk, self.uuid)
+        return computer_str
+
+    def get_label(self, full=True):
```

Comment:
Good catch, the intention was `full=False` as default. Would you agree with this change?

Reply:
Why not have a property like:
```
@property
def full_label(self):
    return '{}@{}'.format(self.label, self.computer.name)
```
That seems a lot cleaner, because I don't see the point of having the method with the argument, if you already have a perfectly good property for `label`

Reply:
Agreed.